sass/components/_post.scss

```diff
@@ -2298,6 +2298,18 @@
 }
 
 #restorePostModal {
+    .modal-body {
+        max-height: calc(100vh - 128px - 61px - 88px);
+
+        .GenericModal__header {
+            padding-top: 0;
+        }
+    }
+
+    .modal-content {
+        padding-top: 44px;
```

Comment:
this was needed to avoid scroller goes behind close button. Basically no change in the UI but shifting padding from header to content